.travis.yml

```diff
@@ -11,6 +11,7 @@ python:
   - 3.5
   - 3.6
   - 3.7
+  - 3.8
```

Comment:
Take out 3.5. This will prepare dev to take Black formatting which requires 3.6 and above.

Reply:
That's what I had suggested back then to provide support to >3.5 only. And why are we going with Black and not YAPF? YAPF will allow for strict formatting as well as presenting a more configurable interface?